incubator/schema-registry/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 name: schema-registry
 home: https://docs.confluent.io/current/schema-registry/docs/index.html
-version: 0.4.5
-appVersion: 4.0.1
+version: 0.4.6
```

Comment:
since some parameters are removed/change name, would it make more sense to change the major version?